fluentui_drawer/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$appCompatVersion"
     implementation "androidx.recyclerview:recyclerview:$recyclerViewVersion"
     implementation "com.google.android.material:material:$materialVersion"
+    implementation 'androidx.activity:activity-compose:1.9.3'
```

Comment:
we have composeActivityVersion constant present in repo, can we use that?

Reply:
Bumped up the compose activity version and added the variable here

Reply:
just double check, if bumping the version will cause any issues with partner teams

Reply:
Will reach out to them

Reply:
It seems to be working fine with older activity versions as well. Will update this.

Reply:
No need for the Compose Activity version upgrade. Tested this down to API26 and it is performing as expected without the Compose Activity version upgrade.